recipe/meta.yaml

```diff
@@ -1,71 +1,163 @@
+      # - libarchive-patches/0005-VC9-compatibility-ignore-some-warnings.patch
+      - libarchive-patches/0006-VC-openssl-define-inline-as-__inline.patch
+      # These warnings are errors when compiling lz-4 support
+      - libarchive-patches/0007-CMake-VC9-compatibility-ignore-some-warnings.patch
+      - libarchive-patches/0008-VC9-compatibility-remove-C99.patch
+      - libarchive-patches/0009-CMake-Force-Multi-threaded-DLL-runtime.patch
+      # 3.4.0 added official support for symlinks on win
+      - libarchive-patches/0010-Use-CreateSymbolicLinkW-on-Windows-for-symlinks-WIP.patch
 
 build:
-  number: 2
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  entry_points:
-    - cph = conda_package_handling.cli:main
-  # HACK: case-sensitive mismatch in latest conda-build
-  missing_dso_whitelist:
-    - "*/VCRUNTIME140.dll"   # [win]
-    - "MSVCR90.dll"   # [win]
+  number: 4
 
 requirements:
   build:
     - {{ compiler('c') }}
-  host:
-    - python
-    - pip
-    - cython
-    - _libarchive_static_for_cph
-    - msinttypes  # [win and vc<14]
-  run:
-    - futures   # [py27]
-    - python
-    - six
-    - tqdm
-
-test:
-  source_files:
-    - tests
-  requires:
-    - mock
-    - pytest
-    - pytest-cov
-    - pytest-mock
-  imports:
-    - conda_package_handling
-    - conda_package_handling.archive_utils_cy
-  commands:
-    # This test is skipped until Dec 2019 which is weird. Skip for now.
-    - pytest tests -k "not test_secure_refusal_to_extract_abs_paths"
+    - {{ compiler('cxx') }}
+    - autoconf    # [unix]
+    - automake    # [unix]
+    - libtool     # [unix]
+    - pkg-config  # [unix]
+    - cmake
+    - make        # [unix]
+    - m2-patch    # [win]
+    - patch       # [unix]
+    - jom         # [win]
+    - ninja       # [win]
+    - m2-gcc-libs  # [win]
+
+outputs:
+  - name: conda-package-handling
+    version: {{ version }}
+    script: cph-build.sh  # [unix]
+    script: cph-bld.bat   # [win]
+    build:
+      entry_points:
+        - cph = conda_package_handling.cli:main
+      # HACK: case-sensitive mismatch in latest conda-build
+      missing_dso_whitelist:
+        - Library/bin/VCRUNTIME140.dll    # [win]
+        - "*/VCRUNTIME140.dll"            # [win]
+        - "*/MSVCR90.dll"                 # [win]
+        - "MSVCR90.dll"                   # [win]
+
+    requirements:
+      build:
+        - {{ compiler('c') }}
+      host:
+        - python
+        - pip
+        - cython
+      run:
+        - python
+        - six
+        - tqdm
+
+    test:
+      source_files:
+        - cph/tests
+      requires:
+        - mock
+        - pytest
+        - pytest-cov
+        - pytest-mock
+      imports:
+        - conda_package_handling
+        - conda_package_handling.archive_utils_cy
+      commands:
+        # This test is skipped until Dec 2019 which is weird. Skip for now.
+        - pytest cph/tests -k "not test_secure_refusal_to_extract_abs_paths"
```

Comment:
We (the cph developers) could not get this test to pass reliably.  Rather than marking as known-fail and likely never revisiting it we put in a time based skip.  

It still has not been fixed so skipping fine,